### ISSUE TITLE
History of recent hospitalization

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10147]
**Preferred term label:**
History of recent hospitalization

**Synonyms**
Nosocomial infection risk

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0205721

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Candida auris infection

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000034
- label: History of recent hospitalization
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A past medical histry of recently being hospitalized.
- comment: This may imply a nosocomial infection risk.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10147
- synonyms:
